xml/System.Text.Unicode/UnicodeRange.xml

```diff
@@ -33,10 +42,25 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="firstCodePoint">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="firstCodePoint">The first code point in the range.</param>
+        <param name="length">The number of code points in the range.</param>
+        <summary>Creates a new <see cref="System.Text.Unicode.UnicodeRange" /> that includes a specified number of characters starting at a specified Unicode code point.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException"> 
+
+           <paramref name="firstCodePoint" /> is less than zero. 
+
+-or-
+           <paramref name="firstCodePoint" /> is greater than 0xFFFF. 
+
+-or-           
+
+<paramref name="length" /> is less than zero.
```

Comment:
We need to add if the length will get us outside the oxFFFF range too.

Reply:
@tarekgh, I've added it.